src/main/java/lotto/view/ResultView.java

```diff
@@ -13,20 +13,17 @@ public class ResultView {
 
     public void printOfBuyLotto(LottoGame lottoGame) {
         System.out.println(lottoGame.countOfLotto() + BUY_LOTTO_MESSAGE);
-        List<Lotto> lottos = lottoGame.getLottos();
-        for (Lotto lotto : lottos) {
-            System.out.println(lotto);
+        for (Lotto lotto : lottoGame.getLottos()) {
+            System.out.println(lotto.sortNumbers());
         }
     }
 
-    public void printWinningResult(LottoGame lottoGame, String winningNumbers) {
-        List<Rank> lottoRanks = lottoGame.match(winningNumbers);
+    public void printWinningResult(LottoGame lottoGame, String winningNumbers, int bonusNumber) {
```

Comment:
```suggestion
    public void printWinningResult(Array LottoResult) {
```

값을 가지고 다니면서 함수를 힘들게 하고 있지 않은가요?